llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -135,6 +135,38 @@ Counter CounterExpressionBuilder::subtract(Counter LHS, Counter RHS,
   return Simplify ? simplify(Cnt) : Cnt;
 }
 
+Counter CounterExpressionBuilder::replace(Counter C, const ReplaceMap &Map) {
```

Comment:
Can this have a more descriptive name? This is replacing each term with an existing term, right?

Reply:
I thought its name may be simple here, since it is declared in the class.

"This replaces each term (or possibly recognizable expression) with an existing **expression**."

Reply:
OK gotcha